packages/client/src/components/SafeSettings/components/EditThreshold/ReviewSafeEdits.js

```diff
@@ -1,58 +1,65 @@
-import SignatureBar from "../SignatureBar";
 const ReviewSafeEdits = ({
   newOwner,
+  ownerToBeRemoved,
   newThreshold,
   allSafeOwners,
   onBack,
   onSubmitClick,
 }) => {
+  const owner = newOwner || ownerToBeRemoved;
   const renderRow = (label, value) => {
     return (
-      <div className="is-flex is-align-items-center is-justify-content-space-between py-5 border-light-top">
-        <label className="has-text-grey">{label}</label>
-        <div className="has-text-black">{value}</div>
+      <div className="columns m-0 py-1  border-light-top">
+        <label className="column px-0 has-text-grey">{label}</label>
+        <div className="column ml-2 px-0 has-text-black has-text-weight-bold">{value}</div>
       </div>
     );
   };
+  const mainSectionHeaderText = () => {
+    if (newOwner) {
+      return "Add Owner"
```

Comment:
I think this should should be New Owner:
https://www.figma.com/file/sKtcz6ytBmNNDNW7XU8ea5/Vessel%3A-User-Experience?node-id=1609%3A237207